Tests/SwiftSyntaxMacroExpansionTest/MacroSystemTests.swift

```diff
@@ -889,30 +857,255 @@ final class MacroSystemTests: XCTestCase {
       var x: Int
       """,
       expandedSource: """
-        var x: Int {
+        var x: Int
+        {
           get {
             _x.wrappedValue
           }
           set {
             _x.wrappedValue = newValue
           }
         }
+
         private var _x: MyWrapperType<Int>
         """,
       macros: testMacros,
       indentationWidth: indentationWidth
     )
   }
 
+  func testAccessorOnVariableDeclWithExistingGetter() {
+    assertMacroExpansion(
+      """
+      @constantOne
+      var x: Int {
```

Comment:
Does the compiler error or replace here?

Reply:
The compiler errors: `variable already has a getter`